controllers/user.ts

```diff
@@ -15,4 +16,27 @@ export class UserController {
         const users = await this.userModel.getAll()
         response.json(users)
     }    
+
+    getById = async (request:Request, response:Response): Promise<void> => {
+      try {
+
+        const userId = parseInt(request.params.id,10)
```

Comment:
One question, what is the use of the 10 after params id?

Reply:
The second params is the base number, in this case,  the base number is 10, another one could  be base 8 (octal) or 16 ( hexadecimal)

Reply:
Ok, understood, thank you very much.